_pages/algorithm-development/your-first-algo.md

```diff
@@ -13,224 +13,215 @@ redirect_from:
+## Publishing Your Algorithm
+The last step is publishing your algorithm. The best part of hosting your code on Algorithmia is that users can access it via an API that takes only a few lines of code to use! Here is what you can set when publishing your algorithm:
 
-On the upper right hand side of the algorithm page you'll see a purple button **Publish** which will start the publish flow:
+On the upper right-hand side of the algorithm page you'll see a purple **Publish** button, which will start the publish flow:
 
 <img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algo_dev_lang/publish_algorithm.png" alt="Publish an algorithm" class="screenshot img-sm">
 
 There are 3 steps to publishing an algorithm: documenting changes, adding example input and output, and configuring algorithm settings.
 
 The **Changes** step shows you your commit history and allows you to add release notes.
 
-The **Example I/O** step is where you'll create your sample input and output for the user to try under Try the API in the Run tab. When you add a sample input, make sure to test it out with all the inputs that you accept since users will be able to test your algorithm with their own inputs.
-
-During the  **Settings** step you can select whether your algorithm will be for public use or private use as well as set the royalty. The algorithm can either be royalty-free or charge per-call. If you opt to have the algorithm charge a royalty, as the author, you will earn 70% of the royalty cost.
-
-Check out [Algorithm Pricing]({{site.baseurl}}/pricing) for more information on how much algorithms will cost to run.
-
-Under "Semantic Version" you can choose which kind of release your change should fall under: Major, Minor, or Revision.
+The **Example Input/Output** step is where you'll enter your sample input, which will be displayed in the Overview tab on the algorithm home page, where users may test your algorithm. This is a good time to test your algorithm with all of the types of inputs that your algorithm accepts, to verify that the output is as expected, because users will be able to test your algorithm with their own inputs.
 
-Under "Pending Changes" you'll see any revisions you've made under the "Manage" tab.
+During the **Configure Settings** step you can select whether your algorithm will be publicly callable or for private use only. Under "Semantic Version" you can choose which kind of release characterizes your changes: Major, Minor, or Revision. Finally, you can enable the collection of metrics using Algorithmia Insights by checking the "Insights" box.
 
-If you are satisfied with your algorithm and settings, go ahead and hit publish. Congratulations, you’re an algorithm developer!
+If you are satisfied with your algorithm code, behavior, and settings, go ahead and hit "Publish". Congratulations, you’re an algorithm developer!
```

Comment:
Again making it a bit more enterprise focused, here let's change "Congratulations, you’re an algorithm developer!" to something like "Congratulations, you just deployed your machine learning model/code into production and it's immediately ready for use!" - or something along those lines

Reply:
yeah I like that, something short: "Congratulations, you just deployed your model into production and it can be called immediately!"

Reply:
cool, feel free to change that and once all these changes are done, I'll approve the PR